config/yurtadm/cloudimage/openyurt-latest/flannel/init-kube.sh

```diff
@@ -0,0 +1,177 @@
+#!/bin/bash
```

Comment:
`init-kube.sh` is used for initializing system for kubernetes, so it's not reasonable to put this file under `flannel` directory.

Reply:
ok